### DISCUSSION TITLE
Renaming Infinite OS

### DISCUSSION THREAD

**Discussion by ntorga** [goinfinite/os#197] [Announcements]
Hey everyone,

We've received feedback that the "OS" in Infinite OS might be causing confusion, as it implies a host OS rather than a container OS. We're committed to being community-driven, so we're opening up the discussion to rename [our open-source shapeshifter/sandbox container operating system](https://github.com/goinfinite/os/).

Our team has brainstormed some potential names, and we'd love to hear your thoughts:

- Sandbox (Abbreviation: SX) - This name fits well with the project's flexibility, as you can shape it into whatever you need post-deployment. We also happen to own the .sx domain, which is a bonus. The SX abbreviation could also stand for Software X, emphasizing the variety of software you can run within it. This is currently our team's top pick.
- Playground (Abbreviation: PG or PD) - Similar to "sandbox," this name conveys the idea of a flexible, adaptable environment.
- Shapeshifter (Abbreviation: SS or SF) - This name highlights the container's ability to transform into whatever you need it to be, whether that's a web app, game server, or something else.
- Chameleon (Abbreviation: CH, CM, or CL) - Like "shapeshifter," this name emphasizes the project's adaptability and ability to change.

We considered "Box" as an option, but it's already been taken by too many other projects and companies, so we didn't include it in the list.

If you have any new name suggestions or just want to vote on one of these options, please share your thoughts. We appreciate your input!

**Comment by andreasfernandes93** [comment]
I like SX

**Comment by alehostert** [comment]
My favorites, in that order:

1. SX
1. SF
1. CH

Curious note: when I see CH, my brain automatically printed a picture of "[El Chapulín Colorado](https://www.imdb.com/pt/title/tt0229888/)" saying one of its catch phrases hahahah

**Comment by luannzin** [comment]
I like SX

**Comment by ArtthurMoraes** [comment]
I prefer SX.

**Comment by felipoantonoff** [comment]
@ntorga Right now, it's going to be Infinite OS, EZ, and BZ, right?
I like the current names and don't think anything related to Sandbox is good for OS, as it could refer to something like a test or a playground.

If I had to choose between the options, it would be Shapeshifter or Chameleon.
Now, even though I'm not so keen on the term Sandbox, SX isn't bad, especially since the X can have a broad meaning, and the S could simply stand for System or something else.

**Reply by alehostert** [reply]
Software X was something we said internally hahahah
Sounds like the software the X-Men would use :rofl:
